src/app/rundown/components/part/part.component.html

```diff
@@ -8,5 +8,5 @@
       <sofie-piece *ngFor="let piece of pieces" [expectedPartDuration]="part.expectedDuration ? part.expectedDuration : DEFAULT_PART_DURATION_IN_MS" [partDuration]="duration" [piece]="piece" [pixelsPerSecond]="pixelsPerSecond"></sofie-piece>
     </div>
   </div>
-  <sofie-timeline-next-indicator [part]="part"></sofie-timeline-next-indicator>
+  <sofie-timeline-flag [label]="part.isNext ? 'NEXT' : undefined" [turnedOn]="part.isNext"></sofie-timeline-flag>
```

Comment:
There is propably a good reason for it, by why the renaming of the component?

Reply:
I think we're open for suggestions here, however, the renaming was due to it serving multiple purposes as either border flags, the actual Next-cursor and the changed look once reaching the last part in a segment.

Reply:
Because the component now does something different, so the name should be updated to reflect what it does now.

Reply:
Previously, it was only used for the next-flag indicator on a part. Now is a bit more generic, such that it can also be used for the end of segment indicator.